2.6.md

```diff
@@ -3032,6 +3064,44 @@ This object allows sellers to specify price floors for video and audio creatives
   </tr>
 </table>
 
+### 3.2.36 - Object: InterestGroupAuctionSupport <b>PROVISIONAL</b> <a name="objectinterestgroupauctionsupport"></a>
+
+This object allows sellers to signal interest group auction support for an Imp. 
+
+<table>
+  <tr>
+    <td><strong>Attribute&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Type&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></td>
+    <td><strong>Description</strong></td>
+  </tr>
+  <tr>
+    <td><code>ae <b>PROVISIONAL</b></code></td>
+    <td>integer;<br>default 1</td>
+    <td>Interest Group auction environment support for this impression: <br>
+1 = on-device orchestrated auction<br>
+2 = server-orchestrated auction<br>
+<br>
+Note that this only indicates that the interest group auction is supported, not that it is guaranteed to execute. If no buyer chooses to participate in the interest group auction, then the interest group auction will be skipped and the winner of the contextual auction, if any, will serve instead.</td>
+  </tr>
+  <tr>
+    <td><code>biddable <b>PROVISIONAL</b></code></td>
```

Comment:
The assumption was that IG bidding would be affirmatively allowed by the seller rather than assumed.